### DISCUSSION TITLE
Projecting to EPSG:6434

### DISCUSSION THREAD

**Discussion by mgracer** [DotSpatial/DotSpatial#1495] [Q&A]
I'm currently using the DotSpatial library to project from web mercator to CT state plane EPSG:2234 using KnownCoordinateSystems.Projected.StatePlaneNad1983Feet.NAD1983StatePlaneConnecticutFIPS0600Feet.  EPSG:2234 has been superceded by EPSG:6434 as the authoritative projection for the State.  Does anyone know if DotSpatial supports projections to EPSG:6434?

Thanks,
- Mike

**Comment by sindizzy** [comment]
If you can get the wkt of the projection you can try it.
